app/livechat/client/views/app/tabbar/visitorForward.html

```diff
@@ -37,7 +37,7 @@ <h3>{{_ "Forward_chat"}}</h3>
 
 				<nav class="buttons">
 					<button class="button button-block cancel" type="button"><span>{{_ "Cancel"}}</span></button>
-					<button class="button button-block primary save"><span>{{_ "Forward"}}</span></button>
+					<button class="button button-block primary save {{forwarded}}" id="forward-livecahat"><span>{{_ "Forward"}}</span></button>
```

Comment:
Please, fix the button id.

```suggestion
					<button class="button button-block primary save {{forwarded}}" id="forward-livechat"><span>{{_ "Forward"}}</span></button>
```